cmd/cvdr/main.go

```diff
@@ -46,7 +46,16 @@ func loadInitialConfig() (*cli.Config, error) {
 				return nil, err
```

Comment:
nit: I would re-order this `if` like this:
```
_, err := os.Stat(path)
if errors.Is(err, os.ErrNotExist) {
  // Import the acloud config
  err = nil
}
if err == nil {
  // Load config file
} else {
  // return error
}
```

That way loading the config file is done in a single place.

Reply:
good point, done